### ISSUE TITLE
"indent" rule `CallExpression` schema is incorrect

### ISSUE THREAD

**Issue by marcfallows** [eslint/eslint#8050]
**Tell us about your environment**

* **ESLint Version:** 3.14.1

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**

```
module.exports = {
  "rules": {
    "indent": [1, 2, {
      "CallExpression": {"arguments": 2}
    }]
  }
};
```

**What actually happened? Please include the actual, raw output from ESLint.**
```
.eslintrc.js:
    Configuration for rule "indent" is invalid:
    Value "data["1"].CallExpression" has additional properties.
```

`indent`'s schema is wrong: https://github.com/eslint/eslint/blob/master/lib/rules/indent.js#L120.
The schema is expecting `parameters` but the code expects `arguments`. ESLint is complaining when I provide `arguments` because it's considered an additional property.

**Comment by not-an-aardvark** [comment]
I think this is a duplicate of https://github.com/eslint/eslint/issues/7612.

Basically, the schema is wrong, but it shouldn't prevent you from creating a valid config.

**Comment by marcfallows** [comment]
But I am getting an error and the schema is setting `additionalProperties: false`. How does my config need to change to be valid?

**Comment by not-an-aardvark** [comment]
The schema is setting `additionalProperties: false` for the top-level object option (i.e. the object with the `CallExpression` key), but the `CallExpression` object does not have `additionalProperties: false`.

The schema in your original comment seems to work for me. Are you sure the schema in the comment matches the one in your config file?

**Comment by marcfallows** [comment]
Closing this, ended up being an unrelated issue relating to an outdated version. Thanks for your help, sorry for wasting your time.

But, it seems crazy to me that the schema is knowingly incorrect. I feel like it would be better to update the schema to remove `parameters` and add `arguments` with the valid values commented out, as opposed to being just plain wrong.

**Comment by not-an-aardvark** [comment]
Yeah, I agree that this is unfortunate. The concern is that this could be a breaking change for users that unknowingly have an invalid value configured for `arguments`.

For what it's worth, this is likely to be fixed in the next major version when https://github.com/eslint/eslint/pull/7618 is merged.
